app/services/foreman_ansible/inventory_creator.rb

```diff
@@ -22,10 +22,14 @@ def to_hash
 
     def host_vars
       hosts.reduce({}) do |hash, host|
-        hash.update(host.fqdn =>
-                    { 'foreman' => host_attributes(host),
+        hash.update(host.fqdn => {
+                      'foreman' => host_attributes(host),
                       'foreman_params' => host_params(host),
-                      'foreman_ansible_roles' => host_roles(host) })
+                      'foreman_ansible_roles' => host_roles(host),
+                      'ansible_port' => host_port(host),
```

Comment:
Please add the old version of these params to ensure compatibility with Ansible 1.x (`ansible_ssh_user', 'ansible_ssh_port') - no harm on sending a few more bytes for this